src/romaine/core.py

```diff
@@ -14,6 +15,7 @@ def __init__(self):
             Initialise Romaine core.
         """
         self.steps = {}
+        self.parser = Parser()
```

Comment:
Maybe it's better to bind a label to the `Parser` class into `Core`'s namespace? That way the class, and its instances, have everything required, without having an instance of the `Parser` hanging around on every copy?